from backend.exception.base_stackable_exception import StackableException


class AuthenticationFailure(StackableException):
    pass
